packages/core/src/components/menu/_submenu.scss

```diff
@@ -17,11 +17,11 @@
       // keep a trail of hovered items as submenus are opened
       /* stylelint-disable scss/at-extend-no-missing-placeholder */
       &:not([class*="#{$ns}-intent-"]) {
-        @extend .#{$ns}-menu-item:hover;
+        @extend .#{$ns}-menu-item, :hover;
```

Comment:
https://sass-lang.com/documentation/at-rules/extend#disallowed-selectors

> If `.message.info` could be extended, the definition of `@extend` says that elements matching the extender would be styled as though they matched `.message.info`. That’s just the same as matching both `.message` and `.info`, so there wouldn’t be any benefit in writing that instead of `@extend .message, .info`.